### ISSUE TITLE
Opposite order of optimizer and lr_scheduler in pytorch 1.1.0+

### ISSUE THREAD

**Issue by kevinchan04** [nerfstudio-project/nerfstudio#1848]
Hi, when I am training nerf in pytorch 1.13 with cuda 11.7, one warning pops out 

> UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule.

Do you have any idea if this warning will affect the NeRF training? Thank you!

**Comment by tancik** [comment]
Can you test #1854 to see if that solves the issue? I'm not seeing the warning locally so I can't confirm that this fixes it.

**Comment by jkulhanek** [comment]
I cannot reproduce the original warning. I was hoping this would fix the error when running nerfacto in torch 2.0.
#1883
